src/modules/pipelines/transform_parquet.bicep

```diff
@@ -0,0 +1,330 @@
+          }
+        ]
+        policy: {
+          timeout: '0.12:00:00'
+          retry: 0
+          retryIntervalInSeconds: 30
+          secureOutput: false
+          secureInput: false
+        }
+        userProperties: []
+        typeProperties: {
+          dataset: {
+            referenceName: sourceDataset
+            type: 'DatasetReference'
+            parameters: {
+              folderName: {
+                value: '@variables(\'destinationFolder\')'
+                type: 'Expression'
+              }
+              fileName: {
+                value: '@variables(\'destinationFile\')'
+                type: 'Expression'
+              }
+            }
+          }
+          enableLogging: false
+          storeSettings: {
+            type: 'AzureBlobFSReadSettings'
+            recursive: true
+            enablePartitionDiscovery: false
+          }
+        }
+      }
+      {
+        name: 'Delete CSV'
+        type: 'Delete'
+        dependsOn: [
+          {
+            activity: 'Convert CSV'
+            dependencyConditions: [
+              'Succeeded'
+            ]
+          }
+        ]
+        policy: {
+          timeout: '0.12:00:00'
+          retry: 0
+          retryIntervalInSeconds: 30
+          secureOutput: false
+          secureInput: false
+        }
+        userProperties: []
+        typeProperties: {
+          dataset: {
+            referenceName: sourceDataset
+            type: 'DatasetReference'
+            parameters: {
+              folderName: {
+                value: '@pipeline().parameters.folderName'
+                type: 'Expression'
+              }
+              fileName: {
+                value: '@pipeline().parameters.fileName'
+                type: 'Expression'
+              }
+            }
+          }
+          enableLogging: false
+          storeSettings: {
+            type: 'AzureBlobFSReadSettings'
+            recursive: true
+            enablePartitionDiscovery: false
+          }
+        }
+      }
+      {
+        name: 'Set Destination Folder Name'
+        type: 'SetVariable'
+        dependsOn: [
+          {
+            activity: 'Set Destination File Name'
+            dependencyConditions: [
+              'Completed'
+            ]
+          }
+        ]
+        userProperties: []
+        typeProperties: {
+          variableName: 'destinationFolder'
+          value: {
+            value: '@replace(concat(variables(\'scope\'),variables(\'date\'),\'/\',variables(\'metric\')),\'//\',\'/\')'
+            type: 'Expression'
+          }
+        }
+      }
+      {
+        name: 'Set Destination File Name'
+        description: ''
+        type: 'SetVariable'
+        dependsOn: [
+          {
+            activity: 'Set Date'
+            dependencyConditions: [
+              'Completed'
+            ]
+          }
+        ]
+        userProperties: []
+        typeProperties: {
+          variableName: 'destinationFile'
```

Comment:
Can we have a single transform pipeline that writes CSV or parquet based on config instead of deployment parameters? Config will be better when we support reconfiguring this in the future.